sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,9 +1,9 @@
+Azure SDK Python packages support for Python 2.7 is ending 01 January 2022. For more information and questions, please refer to https://github.com/Azure/azure-sdk-for-python/issues/20691
```

Comment:
do we add this to changelog or main readme?

Reply:
You are absolutely right!